buildlib/pr/main.yml

```diff
@@ -9,6 +9,9 @@ resources:
     - container: fedora34
       image: rdmz-harbor.rdmz.labs.mlnx/ucx/fedora34:2
       options: --privileged -v /hpc/local:/hpc/local -v /auto/sw_tools:/auto/sw_tools
+    - container: conda
+      image: rdmz-harbor.rdmz.labs.mlnx/ucx/conda:2
+      options: --privileged -v /hpc/local:/hpc/local -v /auto/sw_tools:/auto/sw_tools
```

Comment:
why need --privileged?

Reply:
In the test setup I was running (without Azure), I always had to use `--privileged` for `rc` support. We can remove that if you're sure it will work without it.

Reply:
1. A better way is to pass RDMA devices explicitly: https://community.mellanox.com/s/article/howto-create-docker-container-enabled-with-roce
2. We can start with testing TCP only for now

Reply:
> 1. A better way is to pass RDMA devices explicitly: https://community.mellanox.com/s/article/howto-create-docker-container-enabled-with-roce

How are other tests currently testing with RDMA devices? I couldn't find anything conclusive on how/if test containers indeed have RDMA devices.

> 2. We can start with testing TCP only for now

That's fine if we can't test RDMA, but if we could I'd prefer we do it already.

Reply:
currently there is no test in UCX which runs docker with RDMA device, but our plan is to use the above method to do it (and not --privileged)